infra/README.md

```diff
@@ -0,0 +1,55 @@
+# Meshdb Environment Setup
+
+1. Configure a user for the [proxmox provider](https://registry.terraform.io/providers/Telmate/proxmox/latest/docs) and setup env vars.
+2. Setup tfvars + ssh keys
+3. Create the VMs that will host k3s
+```
+cd meshdb/infra/tf/
+terraform init
+terraform plan --var-file=your.tfvars
+terraform apply --var-file=your.tfvars
+```
```

Comment:
Can you explain what this does? It creates four nodes...? What are the nodes?

Reply:
1 "master"/"control" that is control plane only as well as being used for deployments. Other 3 are worker nodes (running normal pods). There probably don't need to be 3 of them.

Reply:
Right-o, can that be added to the docs?

Reply:
Added